django.utils.autoreload INFO 2023-05-19 23:20:42,343 autoreload 57370 139850082301760 Watching for file changes with StatReloader
django.server INFO 2023-05-19 23:20:44,135 basehttp 57370 139849350772288 "GET / HTTP/1.1" 302 0
django.server INFO 2023-05-19 23:20:44,174 basehttp 57370 139849350772288 "GET /index/ HTTP/1.1" 200 20568
django.server INFO 2023-05-19 23:20:44,256 basehttp 57370 139849350772288 "GET /static/css/custom.css HTTP/1.1" 200 20661
django.server INFO 2023-05-19 23:20:47,271 basehttp 57370 139849350772288 "GET /ddrug/import-vitek/ HTTP/1.1" 200 18956
django.server INFO 2023-05-19 23:20:47,300 basehttp 57370 139849350772288 "GET /static/css/custom.css HTTP/1.1" 304 0
django.server INFO 2023-05-19 23:20:52,052 basehttp 57370 139849350772288 "POST /ddrug/import-vitek/ HTTP/1.1" 200 20305
django.server INFO 2023-05-19 23:20:52,063 basehttp 57370 139849350772288 "GET /static/css/custom.css HTTP/1.1" 304 0
django.request ERROR 2023-05-19 23:21:01,143 log 57370 139849350772288 Internal Server Error: /ddrug/import-vitek/
Traceback (most recent call last):
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/contrib/auth/mixins.py", line 73, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/contrib/auth/mixins.py", line 135, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/formtools/wizard/views.py", line 244, in dispatch
    response = super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/views/generic/base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/formtools/wizard/views.py", line 296, in post
    self.storage.set_step_data(self.steps.current, self.process_step(form))
  File "/home/wz/adjCOADD/ddrug/views.py", line 402, in process_step
    valLog = upload_VitekPDF_List(request, DirName,self.filelist, OrgBatchID=self.orgbatch_id, upload=False)
  File "/home/wz/adjCOADD/ddrug/utils/vitek.py", line 58, in upload_VitekPDF_List
    upload_VitekPDF(DirName,FileList[i],OrgBatchID=OrgBatchID,upload=upload,appuser=Request.user,valLog=valLog)
  File "/home/wz/adjCOADD/ddrug/utils/vitek.py", line 113, in upload_VitekPDF
    djAST = imp_VitekAST_fromDict(c,valLog,upload=upload)
  File "/home/wz/adjCOADD/ddrug/utils/import_drug.py", line 221, in imp_VitekAST_fromDict
    valLog.add_log('Info',iDict['filename'],f"{DrugID.drug_name} ({iDict['bp_source']})",'New [AST] VITEK','-')
AttributeError: 'NoneType' object has no attribute 'drug_name'
django.server ERROR 2023-05-19 23:21:01,145 basehttp 57370 139849350772288 "POST /ddrug/import-vitek/ HTTP/1.1" 500 148594
django.server INFO 2023-05-19 23:21:07,772 basehttp 57370 139849350772288 "GET /ddrug/import-vitek/ HTTP/1.1" 200 18956
django.server INFO 2023-05-19 23:21:07,787 basehttp 57370 139849350772288 "GET /static/css/custom.css HTTP/1.1" 304 0
django.server INFO 2023-05-19 23:21:15,159 basehttp 57370 139849350772288 "POST /ddrug/import-vitek/ HTTP/1.1" 200 20305
django.server INFO 2023-05-19 23:21:15,171 basehttp 57370 139849350772288 "GET /static/css/custom.css HTTP/1.1" 304 0
django.request ERROR 2023-05-19 23:21:29,792 log 57370 139849350772288 Internal Server Error: /ddrug/import-vitek/
Traceback (most recent call last):
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/contrib/auth/mixins.py", line 73, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/contrib/auth/mixins.py", line 135, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/formtools/wizard/views.py", line 244, in dispatch
    response = super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/views/generic/base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/formtools/wizard/views.py", line 296, in post
    self.storage.set_step_data(self.steps.current, self.process_step(form))
  File "/home/wz/adjCOADD/ddrug/views.py", line 402, in process_step
    valLog = upload_VitekPDF_List(request, DirName,self.filelist, OrgBatchID=self.orgbatch_id, upload=False)
  File "/home/wz/adjCOADD/ddrug/utils/vitek.py", line 58, in upload_VitekPDF_List
    upload_VitekPDF(DirName,FileList[i],OrgBatchID=OrgBatchID,upload=upload,appuser=Request.user,valLog=valLog)
  File "/home/wz/adjCOADD/ddrug/utils/vitek.py", line 113, in upload_VitekPDF
    djAST = imp_VitekAST_fromDict(c,valLog,upload=upload)
  File "/home/wz/adjCOADD/ddrug/utils/import_drug.py", line 221, in imp_VitekAST_fromDict
    valLog.add_log('Info',iDict['filename'],f"{DrugID.drug_name} ({iDict['bp_source']})",'New [AST] VITEK','-')
AttributeError: 'NoneType' object has no attribute 'drug_name'
django.server ERROR 2023-05-19 23:21:29,793 basehttp 57370 139849350772288 "POST /ddrug/import-vitek/ HTTP/1.1" 500 148594
django.utils.autoreload INFO 2023-05-19 23:23:44,462 autoreload 57989 139743420331840 Watching for file changes with StatReloader
django.server INFO 2023-05-19 23:23:46,315 basehttp 57989 139742690670144 "GET / HTTP/1.1" 302 0
django.server INFO 2023-05-19 23:23:46,366 basehttp 57989 139742690670144 "GET /index/ HTTP/1.1" 200 20568
django.server INFO 2023-05-19 23:23:46,445 basehttp 57989 139742690670144 "GET /static/css/custom.css HTTP/1.1" 304 0
django.request ERROR 2023-05-19 23:24:29,521 log 57989 139742690670144 Internal Server Error: /ddrug/import-vitek/
Traceback (most recent call last):
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/contrib/auth/mixins.py", line 73, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/contrib/auth/mixins.py", line 135, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/formtools/wizard/views.py", line 244, in dispatch
    response = super().dispatch(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/django/views/generic/base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/wz/anaconda3/envs/coadd_env/lib/python3.10/site-packages/formtools/wizard/views.py", line 296, in post
    self.storage.set_step_data(self.steps.current, self.process_step(form))
  File "/home/wz/adjCOADD/ddrug/views.py", line 402, in process_step
    valLog = upload_VitekPDF_List(request, DirName,self.filelist, OrgBatchID=self.orgbatch_id, upload=False)
  File "/home/wz/adjCOADD/ddrug/utils/vitek.py", line 58, in upload_VitekPDF_List
    upload_VitekPDF(DirName,FileList[i],OrgBatchID=OrgBatchID,upload=upload,appuser=Request.user,valLog=valLog)
  File "/home/wz/adjCOADD/ddrug/utils/vitek.py", line 113, in upload_VitekPDF
    djAST = imp_VitekAST_fromDict(c,valLog,upload=upload)
  File "/home/wz/adjCOADD/ddrug/utils/import_drug.py", line 221, in imp_VitekAST_fromDict
    valLog.add_log('Info',iDict['filename'],f"{DrugID.drug_name} ({iDict['bp_source']})",'New [AST] VITEK','-')
AttributeError: 'NoneType' object has no attribute 'drug_name'
django.server ERROR 2023-05-19 23:24:29,523 basehttp 57989 139742690670144 "POST /ddrug/import-vitek/ HTTP/1.1" 500 148594
django.server INFO 2023-05-19 23:24:36,355 basehttp 57989 139742690670144 "GET /ddrug/import-vitek/ HTTP/1.1" 200 18956
django.server INFO 2023-05-19 23:24:36,370 basehttp 57989 139742690670144 "GET /static/css/custom.css HTTP/1.1" 304 0
django.utils.autoreload INFO 2023-05-19 23:25:19,941 autoreload 57989 139743420331840 /home/wz/adjCOADD/apputil/utils/form_wizard_tools.py changed, reloading.
django.utils.autoreload INFO 2023-05-19 23:25:21,101 autoreload 58335 140255318402880 Watching for file changes with StatReloader
django.server INFO 2023-05-19 23:25:28,072 basehttp 58335 140254586598976 "POST /ddrug/import-vitek/ HTTP/1.1" 200 20328
django.server INFO 2023-05-19 23:25:28,082 basehttp 58335 140254586598976 "GET /static/css/custom.css HTTP/1.1" 304 0
django.server INFO 2023-05-19 23:25:31,504 basehttp 58335 140254586598976 "POST /ddrug/import-vitek/ HTTP/1.1" 200 18956
django.server INFO 2023-05-19 23:25:38,173 basehttp 58335 140254586598976 "POST /ddrug/import-vitek/ HTTP/1.1" 200 21304
django.server INFO 2023-05-19 23:25:40,998 basehttp 58335 140254586598976 "POST /ddrug/import-vitek/ HTTP/1.1" 200 18956
django.server INFO 2023-05-19 23:25:49,231 basehttp 58335 140254586598976 "POST /ddrug/import-vitek/ HTTP/1.1" 200 20328
